### ISSUE TITLE
Command request: onduty - report the current mentor on duty

### ISSUE THREAD

**Issue by craigcabrey** [rit-sse/slushy#1]
It would be awesome to see who is currently on duty. I'm thinking this command should have two modes (besides help):

Anyone:
- `/sse onduty`:
  - `John Doe is currently the mentor on duty.`

Mentors:
- `/sse onduty abc1234`
  - `John Doe has been set to the current mentor on duty.`
  - `John Doe (abc1234) is not a mentor!`
  - `You're not a mentor! You can't change who is currently on duty.`
  - `No such user or mentor abc1234.`

There would be an API endpoint implemented for this: `GET /mentors/current_mentor` and `POST /mentors/current_mentor`.

**Comment by kristenmills** [comment]
Do we actually need `POST /mentors/current_mentor` (which actually should probably be PUT but that's really irrelevant ). `GET /mentors/current_mentor` should just check what time of day it is and the day the week and figure it out.

**Comment by bbesmanoff** [comment]
I like the idea of checking in, if only to help in cases where someone is covering a shift.

Also, it (unofficially) tracks who is holding the lab at a given time.  That idea could also grow to `/sse is-the-lab-open`.

-----Original Message-----

**Comment by kristenmills** [comment]
Fair enough. Maybe we should default to the mentors schedule but check in can override it. That way if someone forgets to check in we have a back up.
